src/sigma.cpp

```diff
@@ -421,15 +428,14 @@ bool CheckSigmaTransaction(
         realHeight = chainActive.Height();
     }
 
-    if(tx.IsSigmaSpend() && realHeight >= consensus.nDisableUnpaddedSigmaBlock && realHeight < consensus.nSigmaPaddingBlock)
-        return state.DoS(100, error("Sigma is bisabled at this period."));
+    bool const allowSigma = (realHeight >= consensus.nSigmaStartBlock);
```

Comment:
Shouldn't we just return immediately if Sigma is not allowed? (I understand that this code already existed)

Reply:
We may be checking the wallet transactions or the block DB transactions. These checks should be done on Sigma disabled block heights.